### PR TITLE
Added pwctl support with check pgwatch3 status (Partial)

### PR DESCRIPTION
**Pull request by pranayjoshi**
This Pull request adds the pwctl support to pgWatch3.

Issues with this pull:

This pull is not yet integrated with the `main()` function. i.e it actually won't work. But the code is locally tested and works as expected. **I am waiting for the maintainers response** to make it fully functional and add the required lines to main() function and to the dockerfile.

Also, please guide me to the contributers guide, if any.

Any form of review and critiques is really appreciated.
Thank you

### CODE REVIEW

File: src/Makefile
Comment (pashagolub) [inline src/Makefile:1]: I don't want makefiles in this projet. We use Go-specific ways, e.g. goreleaser

**Review by pashagolub — COMMENTED** [review]
I'm not sure if we should use `cobra` for such a simple project. I'm afraid it will add extra dependencies. Have you considered other minimalistic ways?

As for folder structure, since pwctl supposed to be standalone binary, this looks promising
```
- cmd
  - pwctl
    - main.go
      *.go
      go.mod
      go.sum
```

**Comment by muni106** [comment]
be careful that your isPortActive function only checks if the port is active, while I think the goal of the "status" command is to check that pgwatch is running on the port

**Comment by pranayjoshi** [comment]
@muni106 Hi please review the full code... IsPortActive is 1 of the 3 functions. Its not even the main function

**Comment by pranayjoshi** [comment]
> I'm not sure if we should use `cobra` for such a simple project. I'm afraid it will add extra dependencies. Have you considered other minimalistic ways?
> 
> As for folder structure, since pwctl supposed to be standalone binary, this looks promising
> 
> ```
> - cmd
>   - pwctl
>     - main.go
>       *.go
>       go.mod
>       go.sum
> ```

@pashagolub  That would make my life so much easier 😄 as now I dont have to deal with clashing with other dependency. Thanks

**Comment by pashagolub** [comment]
No problem! No, of course we can use the existing library. :-) I think there are some lighter. But if you think cobra is the way to go, put it into proposal. I'll gladly consider it as a way to go.

**Comment by pranayjoshi** [comment]
Hey @pashagolub , I researched on some light-weight cli libraries and found `urfave/cli` to be the best in our case. Shall go forward with it?

**Comment by pashagolub** [comment]
It's your task to choose how to proceed. :-) I have no preferences.

**Pull request closed without merging**